rpc/openapi/openapi.yaml

```
@@ -693,6 +693,64 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/ErrorResponse"
+  /header:
+    get:
+      summary: Get the header at a specified height
+      operationId: header
+      parameters:
+        - in: query
+          name: height
+          schema:
+            type: integer
+            default: 0
+            example: 1
+          description: height to return. If no height is provided, it will fetch the latest height.
+      tags:
+        - Info
+      description: |
+        Get Header.
```

I suggest making this a bit more prose-like:
```suggestion
        Retrieve the block header corresponding to a specified height.
```